### DISCUSSION TITLE
...need help with an Artery MCU Gotek

### DISCUSSION THREAD

**Discussion by ChrisDVS** [keirf/flashfloppy#624] [Q&A]
...Hello there ...I hope I'm in the right place to ask for help ??? ...I have recently bought a Gotek to use in an Akai S2000 sampler ...I have flashed it with Flashfloppy ...and everything seemed to install correctly ...but connecting it to the sampler ...nothing happens at all ??? ...I have tried to reflash it a couple of times ...but still nothing ...any help would be greatly received

**Comment by keirf** [comment]
Jumpers set correctly? Does green Gotek LED light at all?

**Comment by ChrisDVS** [comment]
...Hi Keir,

...Thank you for your reply

...So ...I think the correct jumper settings are S1 and JA to J5 ...if I
start the sampler ...the green light flashes ...that's because the sampler
is automatically looking for the OS floppy ...I can start the sampler with
a zip ...and then the Gotek's light is on but very dim

...I still have a working floppy drive ...and that works fine ...I have
tried moving the jumpers to S0 ...both settings with and without JA to J5
...each time the same dim green light

...I used Artery_DFU_DriverInstall and Artery ISP Programmer_V1.5.46 to
flash it with Fashfloppy-v3.28 ...but then I had a popup to download Artery
ISP Programmer_V2.0.02 ...so I tried that too

...As I say ...the driver said it installed OK ...and the same for the
bootloader

...If it helps ...the exact model number is SFRKC30.AT2

...I can't think of more info to give you ...and thank you again for your
help

...Chris

**Comment by keirf** [comment]
You need at least v3.29 firmware to support that Gotek board version. I don't suppose you even have any display output on the Gotek currently! So flash with the latest v3.30 firmware and try again.

There is some documentation on Akai setup too: https://github.com/keirf/flashfloppy/wiki/Host-Platforms#akai

The jumper J5-JA is actually just a spare jumper. It doesn't do anything in that location. You can just remove it. The drive-select jumper is more likely to belong at S0 than S1 but you'll have to try both. Also, note the configuration lines you need in your FF.CFG file.

**Comment by ChrisDVS** [comment]
...Hi again

...Well ...I'm pleased to say ...PROGRESS !!! ...the Gotek's LED is now
showing FF ...I still haven't figured out the FF.CFG file yet ...but it's
all looking good so far ...thank you so much ...you're a star !!!

...Chris

**Reply by keirf** [reply]
It's just a plain text config file in the root of your USB stick. Just put in it the lines recommended in the wiki link I sent above.
```
host=akai
interface=ibmpc-hdout
```
